packages/ses/NEWS.md

```diff
@@ -2,9 +2,13 @@ User-visible changes in SES:
 
 ## Next Release
 
+No changes yet.
```

Comment:
I wouldn't expect to see the "Next Release / No changes yet" section in a released version.

Not a big deal.

Reply:
oh! This PR includes the "back to dev" commit too.